der/CHANGELOG.md

```diff
@@ -4,6 +4,14 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## 0.7.9 (2024-04-01)
+### Changed
+- ignore RUSTSEC-2023-0071 (backport [#1276])
```

Comment:
This is something of a local development configuration change which I'm not sure is worth documenting in the CHANGELOG, and it's also not directly related to `der`, which doesn't use the `rsa` crate

Reply:
Ha well :D 
That went in the 0.7.9 changelog ^^